src/main/java/ap/ims/rest/RestApi.java

```diff
@@ -0,0 +1,18 @@
+package ap.ims.rest;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @TODO Temporary - remove this class.
+ */
+@RestController
+@RequestMapping("/api")
+public class RestApi {
```

Comment:
Temporary.  Should be removed.  Wanted to have a Rest service to prove it wasn't under authentication.  Add I want to get Swagger integrated so I was going to use this to test with.